src/components/Text/Text.tsx

```diff
@@ -22,7 +23,8 @@ export type Props = {
   /**
    * Controls whether to render text inline (defaults to "p");
    */
-  as?: 'p' | 'span';
+  as?: 'p' | 'span' | 'div';
+  capLinelength?: boolean;
```

Comment:
This prop is from the previous `TextPassage`, kept it for sake of existing use cases, let me know how y'all feel

Reply:
I'm just wondering if it should be noted in the component docstring since it's set to `true` by default.

Reply:
I can't leave a comment on the docstring so I'll put it here – I think we should also add in some explanation about how the component works since there's a lot of magic under the hood now.